src/neural_de.jl

```diff
@@ -78,6 +90,14 @@ function (n::NeuralODE{M})(x,p=n.p) where {M<:FastChain}
     solve(prob,n.args...;sensealg=sense,n.kwargs...)
 end
 
+function (n::NeuralODE{M})(x,p=n.p,st=n.st) where {M<:Lux.Chain}
+  dudt_(u,p,t) = n.model(u,p,st)[1]
```

Comment:
This might lead to unexpected results. You want to capture state changes. Something like
```julia
function (n::NeuralODE)(x, ps, st)
    function dudt(u, p, t)
        u_, st = n.model(u, p, st)
        return u_
    end
    prob = ODEProblem{false}(ODEFunction{false}(dudt), x, n.tspan, ps)
    return solve(prob, n.solver; sensealg=n.sensealg, n.kwargs...), st
end
```